byte JOINTS = 1
byte CARTESIAN = 2
byte TARGET = 3
byte type

int64 target
bool lin
float64 a
float64 b
float64 c
float64 x
float64 y
float64 z
float64 a1
float64 a2
float64 a3
float64 a4
float64 a5
float64 a6
float64 a7

int64 base_index
int64 tool_index
float64 velocity
---
string message
